src/physics/radiation/shortwave_radiation.jl

```diff
@@ -28,10 +28,10 @@ end
 initialize!(::TransparentShortwave, ::PrimitiveEquation) = nothing
 
 # function barrier
-parameterization!(ij, diagn, progn, shortwave::TransparentShortwave, model) =
-    shortwave_radiation!(ij, diagn, shortwave, model.planet, model.land_sea_mask)
+@inline parameterization!(ij, diagn, progn, shortwave::TransparentShortwave, model) =
```

Comment:
I haven't `@inline` anything so far, shall we inline everything as a rule of thumb or are there cases where we shouldn't. I'm thinking about the no specialize / CI issues #909 #914 #721 if that logic applies than we should have

```julia
@inline parameterization!(...) = ...
@noinline shortwave_radiation!(...) = ...
```

because otherwise we would overspecialise on `model` in a function that may not depend on all components of model? But still unsure how much that logic actually applies to overall compile time ...